SDWebImage/SDWebImageCodersManager.m

```diff
@@ -13,10 +13,12 @@
 #import "SDWebImageWebPCoder.h"
 #endif
 
+#define LOCK(lock) dispatch_semaphore_wait(lock, DISPATCH_TIME_FOREVER);
+#define UNLOCK(lock) dispatch_semaphore_signal(lock);
```

Comment:
I've already added this to the 5.0.0 release checklist - I mean the fact that those defines are copy pasted to several places. They should probably exist just in the `SDWebImageDefine.h`